src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -29,6 +37,11 @@ trait BackendActor extends Backend with Actor with ActorLogging {
     case CleanUp => cleanUp()
     case SubscribeToEvent(obj) => subscribeToEvent(obj)
     case UnsubscribeToEvent(obj) => unsubscribeToEvent(obj)
-    case ComputeHash => sender ! computeHash
+    case ComputeHash =>
+      val sndr = sender()
+      computeHash onComplete {
```

Comment:
`pipeTo` does not allow for the use of an ADT such as this and wraps failures in a generic `Status.Failure`